### ISSUE TITLE
Is it possible to '\t' for NewStructScanner option?

### ISSUE THREAD

**Issue by six7zero9** [smartystreets/scanners#1]

**Comment by mdwhatcott** [comment]
Yes! Simply pass the result of [`Comma('\t')`](https://godoc.org/github.com/smartystreets/scanners/csv#Comma) to the constructor function:

```
scanner := csv.NewStructScanner(reader, csv.Comma('\t'))
```

Basically, all of the [Option](https://godoc.org/github.com/smartystreets/scanners/csv#Option) funcs correspond to setting a field on the standard lib [*csv.Reader](https://golang.org/pkg/encoding/csv/#Reader).
